.github/actions/validateReassureOutput.js

```diff
@@ -0,0 +1,47 @@
+import * as core from '@actions/core';
```

Comment:
Could this be added in TS already? We are migrating onyx to TS too

Reply:
Sure, I refactored it to TS